### DISCUSSION TITLE
部署方式探讨

### DISCUSSION THREAD

**Discussion by hexWars** [fp-space/MiniNote#3] [General]
将仓库作为模板，其中一个目录存放markdown，markdown里可以写项目使用方式

暂时不部署，但可以写好部署的教程，并放到markdown里，如何

@LordFoxFairy

**Comment by LordFoxFairy** [comment]
可以，我也是这么打算的，笔记下载到固定目录下来后，从里面读取做下渲染

**Reply by hexWars** [reply]
action的你得等一下，我这几天有些别的事情
